app/views/cases/offender_sar/_case_details.html.slim

```diff
@@ -23,10 +23,9 @@
           = render partial: 'cases/sar/third_party_relationship', locals: { case_details: case_details }
           = render partial: 'cases/offender_sar/requester_name', locals: { case_details: case_details }
           = render partial: 'cases/offender_sar/third_party_company_name', locals: { case_details: case_details }
-          = render partial: 'cases/offender_sar/requester_reference', locals: { case_details: case_details }
         = render partial: 'cases/sar/response_address', locals: { case_details: case_details }
-        = render partial: 'cases/offender_sar/recipient_details', locals: { case_details: case_details }
```

Comment:
Did we mean to remove this?

Reply:
I think so - as it has moved from third party recipient dropdown to a new screen it was on the case_details twice if the sending to a third party